docs/source/cuda/overview.rst

```diff
@@ -57,6 +57,31 @@ If you are not using Conda or if you want to use a different version of CUDA
 toolkit, the following describe how Numba searches for a CUDA toolkit
 installation.
 
+.. _cuda-bindings:
+
+CUDA Bindings
+~~~~~~~~~~~~~
+
+Numba supports interacting with the CUDA Driver API via the `NVIDIA CUDA Python
+bindings <https://nvidia.github.io/cuda-python/>`_ and its own ctypes-based
+binding. The ctypes-based binding is presently the default as Per-Thread
+Default Streams and profiler APIs are not supported with the NVIDIA bindings,
+but otherwise functionality is equivalent between the two. You can install the
+NVIDIA bindings with::
+
+   $ conda install nvidia::cuda-python
+
+if you are using Conda, or::
+
+   $ pip install cuda-python
+
+if you are using pip.
+
+The use of NVIDIA bindings is enabled by setting the environment variable
+``NUMBA_CUDA_USE_NVIDIA_BINDING`` to ``"1"``. See
```

Comment:
I think if you use mark up like: ``` :envvar:`NUMBA_CUDA_USE_NVIDIA_BINDING` ``` it creates a link to the env var  doc entry itself if that's useful here?

Reply:
That will be better, I wasn't aware of (or had forgotten) that.